2019/MDAnalysis_ideas_list.md

```diff
@@ -1 +1,84 @@
+__Name:__ MDAnalysis
+
+__Description:__ Scientific Python library for the analysis of molecular dynamics simulations in biophysics, chemistry, materials sciences.
+
+__website:__ https://www.mdanalysis.org/
+
+__repository:__ https://github.com/MDAnalysis/mdanalysis
+
+__email:__ mdnalysis-devel@googlegroups.com
+
+__Project name:__ Make it easy for new users to analyze _their_ data  
+
+__Description:__ MDAnalysis is Python library that provides an abstract and object-oriented interface to data from particle-based simulations (primarily molecular dynamics simulations), which are widely used for simulating diverse systems such as the interaction of drugs with biomolecules or new materials. MDAnalysis is widely used in the scientific community and is written by scientists for scientists. Feedback from our users indicates that they like using MDAnalysis but wished that the documentation were easier to read and had more examples. The docs for [scikit-learn](https://scikit-learn.org/stable/index.html) and the [PyTorch tutorials](https://pytorch.org/tutorials/index.html) are generally cited as excellent examples of documentation and, taking these as examples, we would like to improve our documentation to make it more accessible and more immediately useful for new users. 
+
+We identified four areas for improvement (in rank order of priority):
+
+### Restructure docs
+
+We want to **restructure our docs for user-friendliness** [issue #1175](https://github.com/MDAnalysis/mdanalysis/issues/1175) and refactor docs away from how the source code is organized into how the user interacts with the code (started in [PR #1827](https://github.com/MDAnalysis/mdanalysis/pull/1827)). We envisage a split into three major blocks:
```

Comment:
this is strongly related to the in code documentation mdanalysis has and the other sphinx documentation. It would be good here to point to where documentation is stored, that mdanalysis uses sphinx and that the documentation website is largely auto generated from the code.

Reply:
Will mention.